### PR TITLE
Update Maintainers and Contributors file for overassigning issues

### PR DESCRIPTION
**Pull request by TimOsahenru**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ Follow these steps and note these guidelines to begin contributing:
 
 ## Working on an issue
 
+- As a first-time contributor, please do not request assignment to more than one issue at a time, except in rare scenarios where the issues are closely related.
+- As a returning contributor, ensure that you do not have an open pull request before requesting assignment to a new issue. Your pull request from a previous issue must be reviewed and merged first.
```
Comment (kjaymiller) [inline CONTRIBUTING.md]: I don't think that it's a problem that you work on multiple issues. I think the problem arises when second PR references changes in the first PR...

For instance: 

- A USER edits a file
- The same USER wants to submit another PR but is working on the same branch as the file with changes. 
- The MAINTAINER requests changes on the first PR which now need to be made on the second PR.

**Review by kjaymiller — CHANGES_REQUESTED** [review]
@TimOsahenru great work on this. 

I made a note that needs to be address in the wording around working on multiple PRs at the same time. Please resolve that.

File: CONTRIBUTING.md (hunk 1 above)
Comment (TimOsahenru) [reply CONTRIBUTING.md]: gotcha!

**Comment by TimOsahenru** [comment]
@kjaymiller I've implemented the requested changes, but I get a failing test when I run pytest. I've already pulled the latest changes from upstream. I think I understand the error message, but I'm struggling to find a solution, LOL. Please help. Thanks
image

**Comment by kjaymiller** [comment]
I pulled down changes and I'm not running into that issue.

can you run `pre-commit run -a`?

**Comment by kjaymiller** [comment]
@TimOsahenru  - There may have been a push recently so try another pull of changes. 

I don't think any of that changed the _conferences section.

**Comment by kjaymiller** [comment]
> @kjaymiller I've implemented the requested changes, but I get a failing test when I run pytest. I've already pulled the latest changes from upstream. I think I understand the error message, but I'm struggling to find a solution, LOL. Please help. Thanks image

I ran into a similar issue that I noticed doesn't happen when you do `python -m pytest` I wonder if there is a namespace thing that we're doing.

**Comment by TimOsahenru** [comment]
I've been worked on this bug, still no headway from my end. Previously, I've always ran our test case with `python -m pytest`. After this change https://github.com/BlackPythonDevs/blackpythondevs.github.io/commit/56e317bdd1228ee25ee0f801f928178e0590e29f I noticed that running either `pytest` or `python -m pytest` now causes almost all of our tests to fail

**Comment by kjaymiller** [comment]
@TimOsahenru - I'm not sure as I can't replicate the issue.

**Comment by TimOsahenru** [comment]
> @TimOsahenru - I'm not sure as I can't replicate the issue.

Certainly it's from my end then

**Comment by TimOsahenru** [comment]
is it okay to make a push regardless ?

**Comment by TimOsahenru** [comment]
Finally got the tests to pass by slapping anything to the wall, hoping something sticks:
- Extending kill time by 20
- Adding port number while spinning server  
- Turning off network 😂  
- Running command in CMD instead of my editor's terminal.
Just documenting this here in case anyone else runs into similar issues!

**Comment by kjaymiller** [comment]
Wow that's a lot of things to get going but great troubleshooting. If you continue to have issues we need to look at our setup. There are a lot of things happening. Maybe we can simplify them somehow.

**Review by kjaymiller — APPROVED** [review]

**Comment by TimOsahenru** [comment]
> Wow that's a lot of things to get going but great troubleshooting. If you continue to have issues we need to look at our setup. There are a lot of things happening. Maybe we can simplify them somehow.

agreed
